### ISSUE TITLE
Option to "Opt-out" of Habit's ability to change the value of tasks based on frequency [Suggestion]

### ISSUE THREAD

**Issue by derekantrican** [HabitRPG/habitica#326]
While this is a great feature of habit by default (getting you to do tasks you're not good at for more coins), I think I'd rather it just be one coin for every task no matter what. Or at least not as drastic of a change as it is now (I got .5 coins this morning for completing a task). In fact, if there was no change in tasks for doing them often, but rather only increases in value for doing infrequent ones, that would be great.

EDIT: After thinking more about this, I think it would better be phrased that "tasks that are done frequently, shouldn't be worth less coins". If you think about it, it's a great idea to have tasks that are worth more the more you put them off (so the later you are on completing a task, the more incentive you have to do it). But it's a bad idea for a task to be worth less if you do it often (less incentive to keep doing the task often). If tasks reached a minimum of 1 coin per task, but no less, this would be awesome.

**Comment by horusofoz** [comment]
Though still organic based on completions, would this [Priority Multiplier](https://trello.com/c/4C8w1z5h) system meet your needs?

**Comment by derekantrican** [comment]
Unfortunately, no. The priority system is a great idea but what I'm
basically asking here is that tasks are worth no less than 1 coin. In other
words, being really good at a task shouldn't make it worth less than
"neutral".

Sent from my Nexus 7

**Comment by StanLindsey** [comment]
I kind of agree, my GF has been doing so well recently that she gets hardly any EXP from continuing to do well. If her goal was to level up her character it may be more productive to be actually be unproductive for a while.

I think having a minimal amount is good.

**Comment by derekantrican** [comment]
@SlappyBag Yeah, the XP thing (taking longer to reach each successive level) is already taken care of by making each level cost more XP (i.e. 20 to pass level 2, 60 to pass level 3, etc). If XP is on diminishing returns (less XP for being more successful), this goal is even harder.

Another point is that I base my rewards off of a "1-gold-per-task" mentality. If a task is worth more gold for being unproductive (to encourage productivity), great! I get to my rewards faster for doing older tasks. But because I have been so extreme in my habitRPG system as to include meals in rewards (meals cost 2 gold each), I am foreseeing a future where it takes maybe 5 tasks or more to get one meal (which is not what I have based the reward costs off of).

**Comment by StanLindsey** [comment]
If HabitRPG makes you starve, this is bad times. Great input though thanks. I'll make sure this is brought up.

**Comment by horusofoz** [comment]
@derekantrican I've added the possibility of static minimum rewards for Priority 1 and Priority 2 for reasons similar to your own. With this in mind would it meet your needs?

**Comment by derekantrican** [comment]
@horusofoz My apologies, but could you clarify what you mean by priority 1
& 2?

**Comment by StanLindsey** [comment]
There is a prioritising tasks idea, so you can set a task at priority high, medium or low. If the task is high or medium @horusofoz is suggesting there is a minimum reward for those I believe.

**Comment by derekantrican** [comment]
@horusofoz I was thinking more in terms of a "universal minimum" (e.g. no task is ever worth less than xx gold). I guess relating it to the priority multiplier idea would work, but would it be more work for the user to set minimums in 2 different categories (high and medium) instead of one universally? Additionally, what about a minimum for the "low" priority? I think there would still need to be a minimum for that.

**Comment by horusofoz** [comment]
@derekantrican The minimum for P1/2 would be universal as in P1 can only go so low for all users and the same for P2. You are right that there probably should be a minimum for P3 as well. 

If we enable full user control on rewards, there is likely to some gaming the system, breaking the social planned social aspects as there will be no common point of reference for progress/rewards.

**Comment by derekantrican** [comment]
@horusofoz I agree with not allowing full user control. I would even say that this should be a "behind the scenes" preset that can't be changed by the user. I don't see any reason why the user would want to change the minimum gold they earn other than making it really high. As I mentioned before, 1 gold coin should be a good minimum.

Yes, there should be a minimum for all priority levels. But I don't think I'm understanding why the P3 minimum should be different than the P1/2 minimum. Can you give an example where that might be the preferred case?

**Comment by StanLindsey** [comment]
I think 0.5gp or 0.25gp is a good min but it should only go that far after a long time of repeated success. The idea being that it get less for doing something that has now became natural to do and to concentrate on those tasks you aren't doing well on. Also I feel it isn't too low that completing them becomes irrelevant, it just means that you'd get more experience & gold from forming new habits and continuing to better yourself instead of stagnating and staying static.

**Comment by derekantrican** [comment]
@SlappyBag Ok, I see the logic. I'd agree with .5 gp, but I think .25 gp is really low.

**Comment by wc8** [comment]
Part of it probably depends on how many tasks you have.
Maybe it would help if they lost value more slowly?

**Comment by derekantrican** [comment]
Also true. I would still go with a defined minimum than with a slower loss of value. But here's a new idea that I think will satisfy all: What if the minimum was user-defined, but the value was somewhere between maybe "0.1 and 1 gp" or something like that. So you can't enter something like "25gp" as a minimum. 

@wc8 It could also be based off of tasks somehow. Like the user-defined minimum is somewhere between ".1 and A gp" Where "A" is 1/10 the number of dailys (or something like that).

I think a user-defined minimum is good, but a "bounded user-defined minimum" (with a upper limit on the minimum) prevents people from, as @horusofoz said, "gaming the system" because the user can't put in an excessive minimum.

**Comment by horusofoz** [comment]
@derekantrican Can you procide a summary of the latest iteration of your idea? I will migrate it to Trello once you have it crystallized :)

**Comment by derekantrican** [comment]
@horusofoz Thanks! Here's my best attempt at a summation:

A gold "minimum" should be established for tasks because earning less gold for doing tasks more frequently is actually anti-productive. The minimum gp earned should be somewhere from .5gp-1gp but no less or higher. I propose a minimum earnable gp per task defined in one of two ways:
a) A "universal minimum" that affects everyone. One defined minimum that cannot be changed.
b) A "customizable minimum" where the user can define their own minimum but only within the ".5-1"gp range

Basically this is to allow users to still progress (gp, xp, levels, etc) at a good pace, even if they've been really good at doing their tasks more frequently.

**Comment by PhilipHow** [comment]
Realistically, would anyone not increase the minimum? I guess defining rules for how you want to progress makes sense, but setting values for each user creates a disconnected game.

I say the minimum should be capped to ensure that doing the habit is still worth it, but the degrading ability is a nice feature that makes you pay attention to other habits.

**Comment by derekantrican** [comment]
@MrConcept Understandable. If you are suggesting that we do away with the range of values (.5-1gp) and instead fix it at one point, I suggest the lower of the two: .5gp. It seems that you're also suggesting that there should only be a universal min and not a customizable one, correct?

**Comment by PhilipHow** [comment]
Yeah.

I understand that GTD is a personal thing, so you have to balance customisation with keeping a consistent experience for all the users. It depends where the community wants it to go.

But yeah, I think .5 GP makes it a good enough incentive!

**Comment by horusofoz** [comment]
Apologies for the delayed migration to [Trello](https://trello.com/card/option-opt-out-of-change-reward-value-based-on-frequency/50e5d3684fe3a7266b0036d6/229). Navigate to the card, vote and add your voice to the discussions there.

**Comment by wc8** [comment]
Just a note: discussion on the need to balance rewards to specific work done has been revived [here](http://community.habitrpg.com/content/how-much-gold-coin-worth).

**Comment by derekantrican** [comment]
Thanks!

-Derek

deltamu.com

Philippians 1:18,20-21
